website/static/files/peer_crd_sample.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: clusterlink.net/v1alpha1
```

Comment:
You will need later to update it to include the status.

Reply:
the intent was to show what the user would need to send from yaml to kubectl, not show a full blown CRD response. Status would not be set by the user so it was not noted